apps/codeforafrica/src/payload/collections/Partners.js

```diff
@@ -60,9 +60,22 @@ const Partners = {
       label: "Social Media Links",
       required: false,
     }),
+    {
+      name: "link",
+      type: "group",
+      admin: {
+        hidden: true,
+      },
+      fields: [
+        {
+          name: "href",
+          type: "text",
+          hooks: {
+            beforeValidate: [nestCollectionUnderPage("partners")],
+          },
+        },
+      ],
+    },
   ],
-  hooks: {
-    afterRead: [nestCollectionUnderPage("partners")],
-  },
 };
```

Comment:
Why are we switching from collection hook to field hook? Isn't this breaking other parts that are using nestCollectionUnderPage as collection hook?

Reply:
I still can't figure out why the collection hook crashes the app after sometime.

Reply:
Yes @kilemensi